src/Components/CutPlaneMenu.test.jsx

```diff
@@ -55,5 +56,38 @@ describe('CutPlane', () => {
     expect(callDeletePlanes.length).toBe(1)
     expect(callCreatePlanes.length).toBe(1)
   })
+
+  it('X Section in URL', async () => {
+    render(
+        <ShareMock
+          initialEntries={['/v/p/index.ifc#p:x']}
```

Comment:
related to below.. might as well test the offset here?

Reply:
yes of set is tested.